stdlib/public/core/StaticString.swift

```diff
@@ -131,7 +131,7 @@ public struct StaticString
 
   /// Returns a `String` representing the same sequence of Unicode
   /// scalar values as `self` does.
-  @swift3_migration(message="use the 'String(_:)' initializer")
```

Comment:
Since `StaticString` does not conform to `LosslessLiteralConvertible`, you need to call `String(describing:)`.